conf/database.php

```diff
@@ -0,0 +1,98 @@
+<?php
```

Comment:
Sounds good but I'm not sure to understand where that file is actually used ? Nothing seems to mention it in the diff ?

Reply:
I think I was trying to setup the database with it but with no success as I recall.  For now we have to finish the post-install by setting the database manually.